core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/economy/migrate/MigrationEconomy.java

```diff
@@ -37,15 +37,18 @@
  * @since v1.0.0
  */
 class MigrationEconomy implements EconomyProvider {
-    
-    private static final FailureReason MIGRATION = () ->
-            "The feature is currently not available during migration.";
+
+    private static final FailureReason MIGRATION = () -> "The feature is currently not available during migration.";
```

Comment:
This is too long to be on one line, needs to be reverted to how I had it

Reply:
checkstyle